### PR TITLE
Add documentation for CORS configuration cli option

### PR DESCRIPTION
**Pull request by PtitLuca**
Fixes EDGE-64

This PR adds the documentation for the new CLI option which provides a way to configure the CORS parameters of the JSON-RPC server.

### CODE REVIEW

**Review by zivkovicmilos — APPROVED** [review]
Lookin' good 💯

**Review by lazartravica — CHANGES_REQUESTED** [review]
It is unnecessary to include a new JSON config just to specify what the allowed origins are

File: docs/get-started/cli-commands.mdx [hunk 1]
```diff
@@ -494,6 +494,26 @@ Set block production time in seconds. Default: `2`
...
+
+<Tabs>
+  <TabItem value="syntax" label="Syntax" default>
+
+    server [--access-control-allow-origins ACCESS_CONTROL_ALLOW_ORIGINS]
+
+  </TabItem>
+  <TabItem value="example" label="Example">
+
+    server --access-control-allow-origins "*"
```
Comment (Kourin1996) [inline docs/get-started/cli-commands.mdx:507]: you can specify '*'? what happened if you specify '*'?

**Review by Kourin1996 — CHANGES_REQUESTED** [review]

File: docs/get-started/cli-commands.mdx (hunk 1 above)
Comment (lazartravica) [reply docs/get-started/cli-commands.mdx:507]: @PtitLuca from what I can see here: https://github.com/0xPolygon/polygon-edge/pull/368/files we don't support wildcards?

**Review by lazartravica — CHANGES_REQUESTED** [review]

File: docs/get-started/cli-commands.mdx (hunk 1 above)
Comment (Kourin1996) [reply docs/get-started/cli-commands.mdx:507]: @lazartravica I think so

**Review by Kourin1996 — APPROVED** [review]

**Review by zivkovicmilos — APPROVED** [review]
Lookin' good 💯

**Review by Aleksao998 — APPROVED** [review]
Great job, please resolve merge conflicts.

**Pull request closed without merging**